snooty.toml

```diff
@@ -20,6 +20,8 @@ toc_landing_pages = [
     "/get-started",
```

Comment:
S: update intersphinx: 
https://github.com/mongodb/docs-golang/blob/b671266d4e3364b694ea41687a4c54cfb6ef8116/snooty.toml#L9-L13

Reply:
What does this do? @rustagir